app/controllers/course/achievement/condition/achievements_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Course::Achievement::Condition::AchievementsController <
+  Course::Condition::AchievementsController
+
+  include Course::AchievementConditionalConcern
```

Comment:
Why? Without the mixin, I need to repeat the `conditional=` method in all the condition controllers for a conditional.